Akka.Serialization.MessagePack.Benchmarks/Akka.Serialization.MessagePack.Benchmarks.csproj

```diff
@@ -2,13 +2,13 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
We should probably upgrade this to .NET 7 as part of a larger "modernize build system" effort here